shared/dev/enzymeSnapshotModifiers.ts

```diff
@@ -0,0 +1,19 @@
+// This file contains modifier functions to customize JSON serialization of snapshot tests using enzyme-to-json.
+// Pass these functions to the `map` field in the `option` object enzyme-to-json's toJson.
+// See https://github.com/adriantoine/enzyme-to-json#helper for documentation.
+
+import { Json } from 'enzyme-to-json'
+
+export function replaceHistoryObject(json: Json): Json {
+    if (json.props?.history) {
+        return {
+            ...json,
+            props: {
+                ...json.props,
+                history: 'placeholderHistoryObject',
```

Comment:
This is similar to how the snapshots format `[Function]`
```suggestion
                history: '[History]',
```